modules/swiss/src/main/SwissSheet.scala

```diff
@@ -19,11 +24,21 @@ private object SwissSheet {
   case object Loss    extends Outcome
   case object Draw    extends Outcome
 
-  def pointsFor(outcome: Outcome) =
-    outcome match {
-      case Win | Bye => 2
-      case Draw      => 1
-      case _         => 0
+  def pointsFor(outcome: List[Outcome]) =
+    outcome.foldLeft(0) { case (acc, out) =>
+      acc + (out match {
```

Comment:
I tend to want two methods here, one for the `pointsFor(outcome: Outcome)` which is just the match, and `pointsFor(outcomes: List[Outcome])` which uses the `foldLeft` and the first `pointsFor`